test/functional/feature_connect_coinbase.py

```diff
@@ -31,10 +31,17 @@ def run_test(self):
         assert_equal(node1_info["total_amount"], 50)
 
         coinbase_tx = self.nodes[0].getblock(self.nodes[0].getblockhash(0))["tx"][0]
+        issuance_tx = self.nodes[0].getblock(self.nodes[0].getblockhash(0))["tx"][1]
 
         # Test rpc getraw functionality
-        assert_raises_rpc_error(-5, "The genesis block coinbase is not considered an ordinary transaction and cannot be retrieved", self.nodes[0].getrawtransaction, coinbase_tx)
-        self.nodes[1].getrawtransaction(coinbase_tx)
+
+        # Coinbase transaction is provably unspendable (OP_RETURN), so even AddCoin won't add it
+        assert_raises_rpc_error(-5, "No such mempool transaction. Use -txindex to enable blockchain transaction queries. Use gettransaction for wallet transactions.", self.nodes[0].getrawtransaction, coinbase_tx)
+        assert_raises_rpc_error(-5, "No such mempool transaction. Use -txindex to enable blockchain transaction queries. Use gettransaction for wallet transactions.", self.nodes[1].getrawtransaction, coinbase_tx)
+
+        # Issuance transaction is an OP_TRUE, so will be available to second node
+        assert_raises_rpc_error(-5, "No such mempool transaction. Use -txindex to enable blockchain transaction queries. Use gettransaction for wallet transactions.", self.nodes[0].getrawtransaction, issuance_tx)
+        self.nodes[1].getrawtransaction(issuance_tx)
```

Comment:
Please add a test showing that `listunspent` has a single oiutput worth the amount you set in the conf line: `5000000000`

Reply:
This would require `anyonecanspendaremine=1` correct? Should I add that flag just for this test or shall we make this the default at least for the non-bitcoin functional tests?

Reply:
Oh right, I'd set it just for this test.